systemd/authd.service

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Authd daemon service
+After=authd.socket
+Requires=authd.socket
+PartOf=authd.socket
+
+[Service]
+Type=notify
+ExecStart=/sbin/authd
+
+# Some daemon restrictions
+CapabilityBoundingSet=
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=true
+PrivateDevices=yes
+PrivateMounts=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProcSubset=pid
+ProtectClock=yes
+ProtectControlGroups=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+RestrictAddressFamilies=AF_UNIX
```

Comment:
I think in general from anything particular (like the comments I have above and this one), it could gain some comments, like here "We need to have access to system bus"